src/software/thunderscope/binary_context_managers/game_controller.py

```diff
@@ -27,17 +27,33 @@ class Gamecontroller(object):
     REFEREE_IP = "224.5.23.1"
     CI_MODE_OUTPUT_RECEIVE_BUFFER_SIZE = 9000
 
-    def __init__(self, supress_logs: bool = False) -> None:
+    def __init__(
+        self,
+        supress_logs: bool = False,
+        gamecontroller_port: int = None,
+        referee_addresse: int = None,
```

Comment:
```suggestion
        referee_address: int = None,
```

Reply:
done.

Reply:
bump: looks like it didn't get committed